frontend/src/layouts/SidebarLayout.tsx

```diff
@@ -5,12 +5,19 @@ import { NavigationBar } from "../components/NavigationBar";
 
 export const SidebarLayout = () => {
   return (
-    <Grid container>
-      <Grid item xs={2}>
+    <Grid container wrap="nowrap">
+      <Grid item sx={{ width: "200px" }}>
         <NavigationBar />
       </Grid>
       <Divider orientation="vertical" flexItem sx={{ mr: "-1px" }} />
```

Comment:
The divider seems unnecessary? The negative margin makes it not render on my screen at all.